scene/resources/dynamic_font.cpp

```diff
@@ -677,7 +677,7 @@ DynamicFontAtSize::~DynamicFontAtSize() {
 
 /////////////////////////
 
-void DynamicFont::_reload_cache() {
+void DynamicFont::_reload_cache(const char *p_triggering_property) {
```

Comment:
Any specific reason for `const char *` over `const String &`?

Reply:
Lack of my knowledge about project conventions. `_change_notify` takes in `const char *`, so I reused the type.